content/java/concepts/calendar/terms/add/add.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.add()'
+Description: 'Adds or subtracts a specified amount of time to/from the given Calendar field.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.add()`** method of the `Calendar` class is used to manipulate and modify a `Calendar` instance by adding or subtracting a specified amount of time to/from a specific calendar field.
+
+## Syntax
+
+```pseudo
+myCalendar.add(field, amount)
+```
+
+- `field`: An integer constant representing the field to which the amount of time should be added or subtracted (e.g.,`Calendar.YEAR`, `Calendar.MONTH`, `Calendar.DAY_OF_MONTH`).
+
+- `amount`: An integer value representing the amount of time to add or subtract from the specified field. A positive value adds time, and a negative value subtracts time.
+
+## Example
+
+In the example below, we demonstrate how to use the `.add()` method to manipulate a `Calendar` instance:
```

Comment:
```suggestion
The example below demonstrates how to use the `.add()` method to manipulate a `Calendar` instance:
```

Reply:
I've made the requested edits. Thanks @SSwiniarski!